### PR TITLE
Incremental improvements and fixes to tutorial

### PR DESCRIPTION
**Pull request by mgasner**

### CODE REVIEW

File: python_modules/dagster/dagster/tutorials/intro_tutorial/actual_dag.py [hunk 1]
```diff
@@ -11,33 +11,33 @@ def solid_a():
     return 1
 
 
-@lambda_solid(inputs=[InputDefinition('arg_a')])
+@lambda_solid(inputs=[InputDefinition("arg_a")])
```
Comment (schrockn) [inline python_modules/dagster/dagster/tutorials/intro_tutorial/actual_dag.py]: why switch to double quotes?

File: python_modules/dagster/dagster/tutorials/intro_tutorial/test/test_config.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
```
Comment (schrockn) [inline python_modules/dagster/dagster/tutorials/intro_tutorial/test/test_config.py]: what's the encoding: thing about?

File: python_modules/dagster/docs/intro_tutorial/config.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+We should be able to run the same code in different environments for test, in development, and in
+production, and to parametrize our solids' interactions with the different facilities afforded by
+each of those environments.
+
+Then, we can declaratively specify features of our environment without having to rewrite our code.
+
+Conceptually, where **inputs** are inputs to the computation done by a single solid, and might be
+linked by a dependency definition to **outputs** of a previous computation in a DAG,
+**configuration** should be used to specify *how* a computation executes.
```
Comment (schrockn) [inline python_modules/dagster/docs/intro_tutorial/config.rst]: this is a great explanation 👍🏻

File: python_modules/dagster/docs/intro_tutorial/configuration_schemas.rst [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+And then run the pipeline:
+
+.. code-block:: console
+
+    $ dagster pipeline execute -f configuration_schemas.py \
+    -n define_demo_configuration_schema_pipeline -e configuration_schemas_error_2.yml
+    ...
+    dagster.core.execution.PipelineConfigEvaluationError: Pipeline "demo_configuration_schema" config errors:
+        Error 1: Undefined field "double_the_word_with_typed_config" at path root:solids
```
Comment (schrockn) [inline python_modules/dagster/docs/intro_tutorial/configuration_schemas.rst]: really do ❤️ these error messages

File: python_modules/dagster/dagster/tutorials/intro_tutorial/test/test_config.py (hunk 2 above)
Comment (mgasner) [reply python_modules/dagster/dagster/tutorials/intro_tutorial/test/test_config.py]: py 2.7 compatibility, we could go as deep as we want on this particular horror

Comment (schrockn) [reply python_modules/dagster/dagster/tutorials/intro_tutorial/test/test_config.py]: rgr. comment might be nice

**Pull request merged**